packages/tupaia-web/src/features/DashboardItem/DashboardInfoButton.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { ViewConfig } from '@tupaia/types';
+import MuiInfoIcon from '@material-ui/icons/Info';
+import { MOBILE_BREAKPOINT } from '../../constants';
+
+const OpenDashboardInfoButton = styled.div`
```

Comment:
Can we rename this to just be `Wrapper` or something? It's not actually a button so we should be more descriptive with what it is